client/src/components/account/adminStreamPanel.tsx

```diff
@@ -0,0 +1,272 @@
+import {
+  Call,
+  ParticipantView,
+  useCall,
+  useCallStateHooks,
+} from "@stream-io/video-react-sdk";
+import classNames from "classnames";
+
+import { useEffect, useState } from "react";
+
+/**
+ * This panel fits into the overall AdminPanel and manages the Stream video and audio.
+ */
+const AdminStreamPanel: React.FC<{
+  rtmpsDetails: RtmpsDetails | null;
+}> = ({ rtmpsDetails }) => {
+  const call = useCall();
+  const { useIsCallLive, useParticipants } = useCallStateHooks();
+  const isLive = useIsCallLive();
+  const participants = useParticipants();
+
+  const [streamType, setStreamType] = useState<"RTMPS" | "Browser" | null>(
+    null,
+  );
+
+  const VIDEO = 2;
+  const liveParticipants = participants.filter((p) =>
+    p.publishedTracks.includes(VIDEO),
+  );
+
+  const liveReady = liveParticipants.length > 0;
+
+  console.log({
```

Comment:
Could we remove this comment? It is throwing a log every time!

Reply:
It also re-renders a lot, which worries me, but we can ignore that for now and potentially do another re-render / perf pass once we merge.